src/Components/App/App.js

```diff
@@ -4,27 +4,51 @@ import { fetchData } from '../../data/apiCall';
 import { Home } from '../Home/Home';
 import { Nav } from '../Nav/Nav';
 import './App.css';
+import { ExhibitContainer } from '../ExhibitContainer/ExhibitContainer';
+import { BreedPage } from '../BreedPage/BreedPage';
+import { Redirect, Route, Switch } from "react-router-dom";
 
 const App = () => {
-  const [breed, setBreed] = useState([])
+  const [breed, setBreed] = useState('')
+  const [breedUrls, setBreedUrls] = useState([])
   const [favorites, setFavorites] = useState([])
   const [error, setError] = useState('')
 
   const fetchDogData = () => {
-   fetchData('breed/schnauzer/images')
-    .then(data => console.log(data))
+    console.log('fetch dog')
+  
+    console.log('hi',breed)
+    fetchData(`breed/${breed}/images`)
+    .then(data => {
```

Comment:
I love being able to see your workflow here of testing it out with a hardcoded path then making it dynamic.  Nice work!